ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-release-4.9.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.15
```

Comment:
You'll also want to do release-4.10, since that's what master is feeding now.

Reply:
fixed